### PR TITLE
Hotfix/RTENU-210 restrict workspace for search

### PR DESCRIPTION
**Pull request by NinaDang97**
Restrict search files only in ratat-extra workspace

### CODE REVIEW

File: packages/server/lambdas/alfresco/searchQueryBuilder/index.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ export const searchQueryBuilder = ({
 }: QueryRequest): Query => {
   switch (language) {
     case QueryLanguage.LUCENE:
-      const luceneQueryBuilder = new LuceneQueryBuilder();
+      const luceneQueryBuilder = new LuceneQueryBuilder('/app:company_home/st:sites/cm:ratat-extra//*');
```
Comment (mehtis) [inline packages/server/lambdas/alfresco/searchQueryBuilder/index.ts:14]: Does this work in both dev and prod?

**Review by mehtis — APPROVED** [review]

File: packages/server/lambdas/alfresco/searchQueryBuilder/index.ts (hunk 1 above)
Comment (mehtis) [reply packages/server/lambdas/alfresco/searchQueryBuilder/index.ts:14]: (Accidental approval when I just meant to comment. Revoked my approval for now)

Comment (NinaDang97) [reply packages/server/lambdas/alfresco/searchQueryBuilder/index.ts:14]: Oh true. prod has different site name than dev. Is it good idea to store path name in parameter store?

Comment (mehtis) [reply packages/server/lambdas/alfresco/searchQueryBuilder/index.ts:14]: I'd save it there, yes.

**Comment by NinaDang97** [comment]
Convert to draft cause getting value from parameter store isn't working yet

**Comment by mehtis** [comment]
> Convert to draft cause getting value from parameter store isn't working yet. Got error `1 validation error detected: Value '' at 'name' failed to satisfy constraint: Member must have length greater than or equal to 1`

You need to give rights to the parameter.
Example from an another parameter.
```
    const ssmAlfrescoParameterPolicy = new PolicyStatement({
      actions: ['ssm:GetParameter', 'ssm:GetParameters', 'ssm:DescribeParameters'],
      resources: [`arn:aws:ssm:${this.region}:${this.account}:parameter/${alfrescoAPIKey}`],
    });
```

**Review by mehtis — APPROVED** [review]

**Pull request merged**